ballerina/Module.md

```diff
@@ -1,18 +1,45 @@
-# Module Overview
+# Overview
 
-This module provides relation database support for the `bal persist` feature, which provides functionality to store and query data from a relational database conveniently through a data model instead of using SQL query language.
+This module provides relational database support for the `bal persist` feature, which provides functionality to store and query data from a relational database through a data model instead of writing SQL.
```

Comment:
```suggestion
This package provides relational database support for the `bal persist` feature, which provides functionality to store and query data from a relational database through a data model instead of writing SQL.
```